src/__tests__/ceramic_cas.ts

```diff
@@ -67,4 +67,118 @@ describe('Ceramic<->CAS integration', () => {
         expect(doc.content).toEqual(newContent)
         expect(doc.state.log.length).toEqual(4)
     })
+
+    test("multiple documents, multiple updates per anchor batch", async () => {
```

Comment:
I think we should refactor this to pass a variable `count` that will generate and test `count` number of docs. Very likely we want to increase this number from 4 and it will be nice to do 10, 100, 1000.

Reply:
That sounds more like a load test.  This is just supposed to be a basic sanity check that all the components are integrating properly.  Soon we're going to add more tests, most likely (though not necessarily) within this repo, that will create large numbers of documents and updates.  For those we will probably want more configurable control over how many documents get created, but for this I think a small fixed number is fine

Reply:
Does the number of documents to affect this test though? I'm not clear on the logic of the anchor service batching, but it seems like the behavior maybe be different if we cross some threshold of documents we allow to be batched at once. Does such a threshold exist and can we test for that?

If not agree fixed number is fine but refactoring seems worth it so we can reuse the logic elsewhere. If not no bother.

Reply:
Yeah, there is a limit on number of records in a single anchor batch.  It would indeed be interesting to have a test where we cross that threshold. But to do that we'd need to configure the CAS to run with a lower max batch limit (or create a lot of documents in this test, but I don't think we should do that as it would make the tests take much longer to run).  Either way, I think that could be done later.  I think moving on to the multi-node tests and IDX integration tests is higher priority

Reply:
Ok gotcha. So seems like we need to pass in a variable for what the CAS batch limit is because this test is dependent on what that number is.

Because I think the idea is that we want these tests to be accommodating to different configurations of services. In general I think we should try to think of most of the integration tests in that way, and they should expect to have some configuration handed to them.

I wouldn't worry _too_ much about the time (within reason) because these will be run manually or nightly anyway.

Reply:
For now I'm hoping nightly at least, but longer term I think eventually the goal should be to run these on every commit, same as our unit tests.  The only reason not to do that right away is that these tests will probably be a bit more expensive (in terms of real-world dollars) to run since they may have to spin up nodes on AWS (or equivalent), so that can probably wait until our pockets are a bit deeper